operations/helm/charts/mimir-distributed/templates/smoke-test/smoke-test-job.yaml

```diff
@@ -37,20 +37,24 @@ spec:
         {{- toYaml .Values.smoke_test.initContainers | nindent 8 }}
       containers:
         - name: smoke-test
-          image: "{{ .Values.smoke_test.image.repository }}:{{ .Values.smoke_test.image.tag }}"
-          imagePullPolicy: {{ .Values.smoke_test.image.pullPolicy }}
+          image: "{{ include "mimir.imageReference" . }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
+            - "-target=continuous-test"
+            - "-activity-tracker.filepath=/active-query-tracker/activity.log"
```

Comment:
continuous test doesn't use the tracker, we shouldn't need this config

Reply:
fails without it

Reply:
what fails without it?

Reply:
the binary never starts up because it cannot write to working dir, which is readonly

Reply:
the value here is the same as in the yaml https://github.com/grafana/mimir/blob/03164e899430be7ceabaf643b0a8109bcf460ef8/operations/helm/charts/mimir-distributed/values.yaml#L117-L118

i wonder if it will work if we just set the activity tracker to an empty string, so that it doesn't attempt to start it. I'll give it a try